llpc/util/llpcElfWriter.cpp

```diff
@@ -1143,15 +1143,15 @@ bool getDescriptorOffsetRelocationValue(Context *context, RelocationEntry relocE
   if (isGraphicsPipeline) {
     auto pPipelineInfo = reinterpret_cast<const GraphicsPipelineBuildInfo *>(context->getPipelineBuildInfo());
```

Comment:
I see you have made a change to correct the coding style issue. And you reverted it because of the backward compatibility problem. I think the function-internal naming is still safe to be corrected like pPipelineInfo -> pipelineInfo. It could be another PR in order to not distract the audience from this one.

Reply:
Thanks Rex. Shall we leave that to another PR? Are you able to approve this one please?

Reply:
I'm OK with making another PR to correct this. I can approve this if no one has additional comments.